docs/src/components/HeaderBox.tsx

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from "styled-components";
+import { mediaQueries as mq } from "@kiwicom/orbit-components";
+
+import { MAX_CONTENT_WIDTH } from "../consts";
+
+const HeaderBox = styled.div`
+  ${mq.desktop(css`
+    position: relative;
+    height: 100vh;
+    padding: 64px;
+    background: linear-gradient(80.68deg, #f1f4f7 7.05%, #f5f7f9 57.05%);
+    z-index: 0;
```

Comment:
could you place mq query below the normal CSS code? With mobile-first approach it's better to read it top-down ;)